addons/website_event_track/static/src/js/website_event_track.js

```diff
@@ -55,6 +56,14 @@ publicWidget.registry.websiteEventTrack = publicWidget.Widget.extend({
                     this._updateAgendaScroll();
                 }
             }
+            this.listItems = document.querySelectorAll('.o_wesession_list_item')
```

Comment:
Ah yes it occured to me, makes more sense Thanks for the clarification :)

Reply:
But in desktop mode we have clickable tags on the card as well ?

Reply:
We don't need to use stopPropagation since the tags are simply data-post redirecting with filtering. We can wrap the o_wesession_list_item directly in an <a> tag, and the tags will still work correctly. What do you think? :)

Reply:
I think we should keep the code as simple and standard as possible. If it's simpler, why not?

For the talk link, I prefer having a standard `<a href="..."`>`  to enable Google, Bing, etc crawlers to index the talk.

If I remember well, we used span for the tags to prevent the web crawlers from exploring those links and cycling.